### ISSUE TITLE
Add Skip Link Focus

### ISSUE THREAD

**Issue by spencerfinnell** [bigboxwc/bigbox#90]
### Expected Behavior

Should be a link to skip to page content for keyboard users.

### Actual Behavior

There is not.

--

https://github.com/WordPress/twentynineteen/pull/47
